modules/learn/pages/clusters-and-availability/groups.adoc

```diff
@@ -201,9 +212,4 @@ To enable the failover of all nodes in a group:
 * With CLI, see xref:cli:cbcli/couchbase-cli-setting-autofailover.adoc[setting-autofailover].
 * With the REST API, see xref:rest-api:rest-cluster-autofailover-enable.adoc[Enabling and Disabling Auto-Failover].
 
-Note that in 7.1+, _automatic failover_ can fail over more than three nodes concurrently: this has permitted the removal of pre-7.1 interfaces that were specific to triggering _auto-failover for server groups_.
-Consequently, in order to ensure successful auto-failover of a server group, the maximum count for auto-failover must be established by the administrator as a value equal to or greater than the number of nodes in the server group.
 
-Note that the pre-7.1 interfaces for triggering auto-failover for server groups have been _removed_ from 7.1: therefore, programs that attempt to use the pre-7.1 interfaces with 7.1+ will _fail_.
```

Comment:
Lines 207-209 should be kept.   I realize I had previously said that they should be removed, but looking at the information in this section again, I see that it's needed.  Also, users may navigate to this section from the right-hand menu without looking at the rest of the page.  So, this info is needed and appropriate here.